pywinauto/unittests/test_application_macos.py

```diff
@@ -31,18 +31,11 @@ def _test_app():
 sys.path.append(".")
 
 if sys.platform == 'darwin':
-    class ApplicationTestCases(unittest.TestCase):
-
-        """Unit tests for the application.Application class"""
-
-        def setUp(self):
-
-            pass
-
-        def tearDown(self):
-
-            pass
-
+    class ApplicationEspecialTestCases(unittest.TestCase):
```

Comment:
A small suggestion, so if these tests for start/end, let's than call it `ApplicationStartEndTestCases`.